testdata/test100result/result.go

```diff
@@ -0,0 +1,75 @@
+package testdata
```

Comment:
See my comment above about renaming this file and putting it in the goldens directory.